test/conformance/kuttl/kuttl-test.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestSuite
 testDirs:
-- ./test/conformance/kuttl
+- ./test/conformance/kuttl/events/clusterpolicy/generate-events-upon-successful-generation
```

```suggestion
- ./test/conformance/kuttl
```

IIRC this is not used anymore.